package.json

```diff
@@ -90,7 +90,7 @@
     "@nomiclabs/hardhat-ethers": "^2.2.1",
     "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client",
     "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts",
-    "@rsksmart/rif-relay-server": "github:rsksmart/rif-relay-server",
+    "@rsksmart/rif-relay-server": "github:rsksmart/rif-relay-server#PP-842/fix-underpriced-transactions",
```

Comment:
This needs to be updated when the [related PR](https://github.com/rsksmart/rif-relay-server/pull/128) is merged.